bftengine/src/bftengine/messages/MsgsCertificate.hpp

```diff
@@ -105,7 +108,7 @@ MsgsCertificate<T, SelfTrust, SelfIsRequired, KeepAllMsgs, ExternalFunc>::MsgsCe
                                                                                           const uint16_t numOfRequired,
                                                                                           const ReplicaId selfReplicaId)
     : numOfReps{numOfReplicas}, maxFails{maxFailures}, required{numOfRequired}, selfId(selfReplicaId) {
-  static_assert(KeepAllMsgs, "KeepAllMsgs==false is not supported yet");
+  //  static_assert(KeepAllMsgs, "KeepAllMsgs==false is not supported yet");
```

Comment:
Let's just remove this line rather than commenting it out please.

Reply:
doen